CHANGELOG_PENDING.md

```diff
@@ -94,6 +94,7 @@ Friendly reminder: We have a [bug bounty program](https://hackerone.com/tendermi
 - [statesync] \#6378 Retry requests for snapshots and add a minimum discovery time (5s) for new snapshots.
 - [node/state] \#6370 graceful shutdown in the consensus reactor (@JayT106)
 - [crypto/merkle] \#6443 Improve HashAlternatives performance (@cuonglm)
+- [state/indexer] \#6382 reconstruct indexer, move txindex into the indexer package (@JayT106)
```

Comment:
I think this _technically_ a breaking change? Right @marbar3778?

Reply:
Best to treat it as such because its in the public API